clients/vscode-hlasmplugin/src/hlasmOutputContentProvider.ts

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (c) 2024 Broadcom.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Broadcom, Inc. - initial API and implementation
+ */
+
+import * as vscode from 'vscode';
+import * as vscodelc from 'vscode-languageclient';
+import { uriFriendlyBase16Decode, uriFriendlyBase16Encode } from './conversions';
+import { isCancellationError } from './helpers';
+import { pickUser } from './uiUtils';
+
+type OutputResult = {
+    level: number;
+    text: string;
+}[];
+
+type Options = { mnote: true, punch: true } | { mnote: false, punch: true } | { mnote: true, punch: false };
+
+const scheme = 'hlasm-output';
```

Comment:
I don't remember - do we already store individual schemes somewhere in "scheme storage utility"? And if we don't, does it make sense to start doing now?